setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-__version__ = '0.17.30'
+__version__ = '0.18.0'
```

Comment:
It's not a breaking change... let's keep it at 17.

```suggestion
__version__ = '0.17.31'
```

Reply:
but, if you want to use data loading, you need a min version. That is why i was thinking of upping the minor?

Reply:
hmmm if you want to use data loading, you must be on `0.17.31+`, same thing, right? we should really move to v1 :)

Reply:
ya soon :)